src/core/Akka.FSharp/Akka.FSharp.fsproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <AssemblyTitle>Akka.FSharp</AssemblyTitle>
     <Description>F# API support for Akka.NET</Description>
-    <TargetFrameworks>net45</TargetFrameworks>
+    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
```

Comment:
Just for clarification - is bumping from .NET 4.5 to 4.5.2 necessary here?

Reply:
No, will revert that